src/js/components/VoterGuide/OrganizationCard.jsx

```diff
@@ -1,295 +1,325 @@
-                  linkIdAttribute="organizationTwitterHandle"
-                  url={`https://twitter.com/${organizationTwitterHandle}`}
-                  target="_blank"
-                  body={(
-                    <span>
-                      <TwitterHandleWrapper>
-                        @
-                        {organizationTwitterHandle}
-                      </TwitterHandleWrapper>
-                      {(twitterFollowersCount && String(twitterFollowersCount) !== '0' && numberWithCommas(twitterFollowersCount) !== '0') && (
-                        <span className="twitter-followers__badge">
-                          {/* <TwitterFollowersIcon className="fab fa-twitter" /> */}
-                          <Twitter />
-                          {numberWithCommas(twitterFollowersCount)}
-                        </span>
-                      )}
-                    </span>
-                  )}
-                />
-              )}
-              { organizationWebsite && (
-                <WebsiteWrapper>
-                  <OpenExternalWebSite
-                    linkIdAttribute="organizationWebsite"
-                    url={organizationWebsite}
-                    target="_blank"
-                    body={(
-                      <span className="u-no-break">
-                        Website
-                        {' '}
-                        <ExternalLinkIcon />
-                      </span>
-                    )}
-                  />
-                </WebsiteWrapper>
-              )}
-              <IssuesByOrganizationDisplayList
-                organizationWeVoteId={this.state.organizationWeVoteId}
-                placement="bottom"
-              />
-              {/* 5 of your friends follow Organization Name<br /> */}
-            </div>
-          ) : null}
-          { this.state.organizationPosition.vote_smart_rating ? (
-            <RatingPopover
-              showDescription={this.state.showRatingDescription}
-              toggleDescription={this.toggleRatingDescription}
-            />
-          ) : null}
-        </div>
-      </OrganizationCardWrapper>
-    );
-  }
-}
-OrganizationCard.propTypes = {
-  ballotItemWeVoteId: PropTypes.string,
-  currentBallotIdInUrl: PropTypes.string,
-  followToggleOn: PropTypes.bool,
-  organization: PropTypes.object.isRequired,
-  turnOffDescription: PropTypes.bool,
-  turnOffLogo: PropTypes.bool,
-  turnOffTwitterHandle: PropTypes.bool,
-  useReadMoreForTwitterDescription: PropTypes.bool,
-  urlWithoutHash: PropTypes.string,
-  we_vote_id: PropTypes.string,
-};
-
-const OrganizationCardWrapper = styled.div`
-`;
-
-// const TwitterFollowersIcon = styled.span`
-//   font-size: 1.25rem;
-//   color: #ccc;
-//   margin-right: 2px;
-//   vertical-align: bottom;
-// `;
-
-const TwitterHandleWrapper = styled.span`
-  margin-right: 10px;
-`;
-
-const WebsiteWrapper = styled.div`
-  margin-left: 4px;
-`;
+import { Twitter } from '@material-ui/icons';

```

Comment:
Thank you @flopmingo for your pull request! I'm noticing that this entire file is showing every line has changed, and I wonder if it might be different line breaks in your updated file, or if it was deleted and then replaced? This doesn't have to stop the pull request, but I just note that it is odd in case you have any ideas.

Reply:
Thanks, @DaleMcGrew .  We will look into this further and get back to you.  Thanks for being so supportive and for allowing us the opportunity to contribute to your project :)

Reply:
Sorry for the late reply! We were working on the code through VS code and I believe it may have been because we changed the EOL sequences from CRLF to LF through VS code. As @mgg143 has said, thank you for giving us this learning opportunity!

Reply:
Thank you for doing the research, and thank you for contributing to We Vote! Could you please update your pull request with the line breaks set back to CRLF? That way we'll be able to see all of the actual changes more easily.

Reply:
Hello Dale! I have been attempting to revert back to an older version of the file by using git checkout file. However, when I revert back to an older version in VS code the line breaks are still set as LF. I'm afraid that simply just changing the files back to CRLF will still highlight every line as a change, thus still making it difficult to see the actual changes. Should I change the line breaks on my current file and update the PR or should I continue to try to find a way to revert back to an older version?

Reply:
I suspect VS code has a setting to change the default line breaks to CRLF, so I would focus on that solution. I would suggest a Google search for "visual studio crlf git". This is interesting:
https://developercommunity.visualstudio.com/t/git-detects-line-endings-as-changes/807495

If there isn't a solution in that, it might be fastest to revert the file and then make the changes again from scratch.
PS. I'm not a fan of Visual Studio for open source projects. I recommend the JetBrains tools, and more specifically WebStorm which offers a free student license.